stdlib/public/SDK/CoreGraphics/CoreGraphics.swift

```diff
@@ -49,14 +49,34 @@ extension CGColor {
 }
 
 public protocol _CGColorInitTrampoline {
-  init(red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat)
+  init?(colorSpace space: CGColorSpace, components: UnsafePointer<CGFloat>)
 }
 
 extension _CGColorInitTrampoline {
   public init(_colorLiteralRed red: Float, green: Float, blue: Float,
               alpha: Float) {
-    self.init(red: CGFloat(red), green: CGFloat(green), blue: CGFloat(blue),
-              alpha: CGFloat(alpha))
+    // This initializer used to call the CGColorCreateGenericRGB, which is
+    // known to Swift as CGColor(red:green:blue:alpha:). Unfortunately this API
+    // is not available on platforms other than macOS. It would be possible to
+    // replicate the exact functionality of that API using
+    // kGColorSpaceGenericRGB, but it is marked as unavailable for Swift. The
```

Comment:
I don't think we should change the behavior on macOS, though; that should continue using the generic `init(red:green:blue:alpha:)`.

Reply:
(unless the CoreGraphics folks want it to be consistent across platforms)

Reply:
Yeah, knowing nothing about color spaces etc., I asked them for a review. Calling the old one for macOS is a good idea, and I can even do that with `if #available`, thanks!

Reply:
Ah, `#available` won't do that, since it treats platforms you don't mention as "true". You'll still want `#if`.

Reply:
😞